src/main/java/de/tum/in/www1/artemis/web/rest/UserJWTController.java

```diff
@@ -83,16 +84,16 @@ public ResponseEntity<JWTToken> authorize(@Valid @RequestBody LoginVM loginVM, @
     /**
      * Authorizes a User logged in with SAML2
      *
-     * @param body the body of the request. "true" to remember the user.
-     * @return a JWT Token if the authorization is successful
+     * @param rememberMe    the body of the request. true to remember the user.
+     * @param response      HTTP response
+     * @return the ResponseEntity with status 200 (ok), 401 (unauthorized) or 403 (user not activated)
      */
     @PostMapping("/saml2")
-    public ResponseEntity<JWTToken> authorizeSAML2(@RequestBody final String body) {
+    public ResponseEntity<Void> authorizeSAML2(@RequestBody final boolean rememberMe, HttpServletResponse response) {
         if (saml2Service.isEmpty()) {
             throw new AccessForbiddenException("SAML2 is disabled");
         }
 
-        final boolean rememberMe = Boolean.parseBoolean(body);
         Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
```

Comment:
This change leads to a HTTP 415 Unsupported Media Type
![image](https://user-images.githubusercontent.com/22198798/203002534-82d9ed03-e0f2-4558-997d-306ab2d2b50d.png)

Reply:
Should be fixed in [6879acd](https://github.com/ls1intum/Artemis/pull/5723/commits/6879acd725d85de8b8fc4105e2702660e507f152)